css/style.css

```diff
@@ -4,6 +4,54 @@
 }
 
 /* ################ (bad) darkmode from @etsde ################ */
+
+@font-face {
+    font-family: 'stolzl-bold';
+    src: url('/font/stolzl.bold.ttf') format('truetype'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+
+@font-face {
+    font-family: 'stolzl-book';
+    src: url('/font/stolzl.book.ttf') format('truetype');
+}
+
+@font-face {
+    font-family: 'stolzl-light';
+    src: url('/font/stolzl.light.ttf') format('truetype');
+}
+
+/* TODO: WOFF files - uncomment and add when available
+@font-face {
+    font-family: 'stolzl-bold';
+    src: url('/font/stolzl.bold.woff') format('woff'); /* Chrome 4+, Firefox 3.5, Opera 10+, Safari 3—5 
+}
+
+@font-face {
+    font-family: 'stolzl-book';
+    src: url('/font/stolzl.book.woff') format('woff');
+}
+
+@font-face {
+    font-family: 'stolzl-light';
+    src: url('/font/stolzl.light.woff') format('woff');
+}
+*/
+
+* {
+    font-family: 'stolzl-book';
+}
+
+/*  Would not use differnet font styles, book only in different sizes is enough in my opinion. 
+    The following just does not look good.
+
+.weekday {
+    font-family: 'stolzl-bold';
+}
+*/
```

Comment:
Can this be removed then?

Reply:
wanted to await @h3yFlo's thoughts

Reply:
I would just go with it, I like it (so remove this).